apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1557,7 +1557,8 @@ expression: "&schema"
       "description": "Health check configuration",
       "default": {
         "listen": "127.0.0.1:8088",
-        "enabled": true
+        "enabled": true,
+        "endpoint": "/health"
```

Comment:
Call it path to mirror the listen config? https://www.apollographql.com/docs/router/configuration/overview

Reply:
done!